lib/renuo_bin_check/script_config.rb

```diff
@@ -53,8 +58,10 @@ def appended_error_output
     end
 
     def script_command
-      raise 'There must be a command set for each script you want to run. Find further instruction on how to use this' \
```

Comment:
no longer so sure after having a look at it: since this extracted method would be only used once, you would still pack it in a method? my general rule is, that i extract as soon as i use it twice